.gitmodules

```diff
@@ -8,3 +8,6 @@
 [submodule "lib/ahrs"]
 	path = lib/ahrs
 	url = https://github.com/Mayitzin/ahrs.git
+[submodule "lib/optical-pile-tracking"]
```

Comment:
This is not how I imagined the submodule would be included. I thought we would include the opt dir directly as msb/opt.
What is the benefit of including opt this way, instead of cloning it externally, if we need to use symlinks, anyway?

Reply:
fair point, the benefit would be that the optical pile tracking repository would always be located at the same (relative) location regarding the other modules, allowing automatic setup of symlinks. Plus, as a submodule, the checked out version is also defined, which would make sure, that the correct version of the optical pile tracking repository is checked out.

Reply:
If we automate the creation of the symlink, we might as well automate the download of the opt. This would also allow a definition of a specific version. If we do not restructure the opt repository, so that we can directly include it at msb/opt, I vote to not inlcude it at all.

Reply:
alright, fair enough. I removed optical pile tracking as a submodule.